<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_canvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_draw_obj">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>tx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>777</x>
      <y>190</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>tx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_translate">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_sx">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_tx">
    <property name="geometry">
     <rect>
      <x>813</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>tx</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_ra">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>777</x>
      <y>240</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>tx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scale">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_sy">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rotate">
    <property name="geometry">
     <rect>
      <x>769</x>
      <y>420</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_ty">
    <property name="geometry">
     <rect>
      <x>813</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
